pkg/config/http.go

```diff
@@ -70,7 +69,7 @@ func (gl *HttpLoader) Run(ctx context.Context) {
 		gl.cCfgChecks <- runtimeCfg.Checks
 
 		timer := time.NewTimer(gl.cfg.Loader.Interval)
-		defer timer.Stop()
+		defer timer.Stop() //nolint:gocritic // TODO: check if this is right
```

Comment:
This will be part of another PR for #35?

Reply:
right!